src/form/fields/FieldView.js

```diff
@@ -198,7 +195,7 @@ export default Marionette.LayoutView.extend({
         let ConstructorFn = this.schema.type;
         this.editor = new ConstructorFn({
             schema: this.schema,
-            form: this.form,
+            form: this.form || this,

```

Comment:
Let's not do this. We always must provide a form through the field `options`, otherwise it's null.